tests/functional/includes/date.mock.js

```diff
@@ -0,0 +1,14 @@
+// Highjack the Date object so that our tests don't suddenly start failing...
```

Comment:
I'm sorry, I misunderstood. The splash example is actually not a problem. I can pass in a static value (right now, I pass it using `Date.now()`, which invokes my mock, but I can just return a static value) Regardless, the mock object is still needed. See thread at the bottom of this PR.